src/bin/facilitator-init.ts

```diff
@@ -133,10 +133,18 @@ commander
           {
             originChainId,
             gatewayAddresses,
-          } = FacilitatorInit.getFromMosaicConfig(auxChainId, options.mosaicConfig)
+          } = FacilitatorInit.getFromGatewayConfig(auxChainId, options.gatewayConfig)
```

Comment:
Please add unit test case for this.

Reply:
As discussed, as it is bin file and it needs refactoring before we write unit test-cases. Ticket https://github.com/mosaicdao/facilitator/issues/240 has been created to refactor code for faciltiator-init binary.